### PR TITLE
feat: add PWD button

### PR DESCRIPTION
**Pull request by revant**
- add pwd button in readme
- add workflow to update pwd.yml

Test:

<a href="https://labs.play-with-docker.com/?stack=https://raw.githubusercontent.com/revant/frappe_docker/pwd/pwd.yml">
  Try in PWD
</a>

Wait for 5 minutes for ERPNext site to be created or check `create-site` container logs before opening browser on port 8080. (username: `Administrator`, password: `admin`)

### CODE REVIEW

File: .github/workflows/build_stable.yml [hunk 1]
```diff
@@ -75,18 +75,20 @@ jobs:
         run: python3 ./.github/scripts/get_latest_tags.py --repo erpnext --version 13
 
       - name: Update
-        run: python3 ./.github/scripts/update_example_env.py
+        run: |
```
Comment (vrslev) [inline .github/workflows/build_stable.yml:78]: Change step name from `update_example_env` to more generic one

Comment (revant) [reply .github/workflows/build_stable.yml:78]: reuse same function?

Comment (vrslev) [reply .github/workflows/build_stable.yml:78]: Just the name: https://github.com/frappe/frappe_docker/blob/dfd98dd1216dda1241b31743639a118ecf8b456d/.github/workflows/build_stable.yml#L60

Comment (revant) [reply .github/workflows/build_stable.yml:78]: updated

**Review by vrslev — APPROVED** [review]

**Pull request merged**